nvflare/lighter/utils.py

```diff
@@ -181,12 +181,40 @@ def sign_all(content_folder, signing_pri_key):
 
 
 def load_yaml(file):
+
+    root = os.path.split(file)[0]
+    yaml_data = None
     if isinstance(file, str):
-        return yaml.safe_load(open(file, "r"))
+        yaml_data = yaml.safe_load(open(file, "r"))
     elif isinstance(file, bytes):
-        return yaml.safe_load(file)
-    else:
-        return None
+        yaml_data = yaml.safe_load(file)
+
+    yaml_data = load_yaml_include(root, yaml_data)
+
+    return yaml_data
+
+
+def load_yaml_include(root, yaml_data):
+    new_data = {}
+    for k, v in yaml_data.items():
+        if k == "include":
+            if isinstance(v, str):
+                includes = [v]
+            elif isinstance(v, list):
+                includes = v
+            for item in includes:
+                new_data.update(load_yaml(os.path.join(root, item)))
+        elif isinstance(v, list):
+            new_list = []
+            for item in v:
+                if isinstance(item, dict):
+                    item = load_yaml_include(root, item)
+                new_list.append(item)
+            new_data[k] = new_list
+        else:
```

Comment:
v could be a dict that contains "include" too. For completeness, should do load_yaml_include(root, v).

Reply:
Added dict value handling. Also added the unit test case.